docs-examples/example-kotlin/src/test/kotlin/io/micronaut/rabbitmq/docs/event/MyStartedEventListener.kt

```diff
@@ -0,0 +1,18 @@
+package io.micronaut.rabbitmq.docs.event
+
+import io.micronaut.context.annotation.Requires
+// tag::imports[]
+import io.micronaut.context.event.ApplicationEventListener
+import io.micronaut.rabbitmq.event.RabbitConsumerStarted
+import jakarta.inject.Singleton
+// end::imports[]
+
+@Requires(property = "spec.name", value = "RabbitListenerEventsSpec")
+// tag::clazz[]
+@Singleton
+class MyStartedEventListener : ApplicationEventListener<RabbitConsumerStarted> {
+  override fun onApplicationEvent(event: RabbitConsumerStarted) {
+    println("RabbitMQ consumer: ${event.source} (method: ${event.method}) just subscribed to: ${event.queue}")
```

Comment:
use `LOG` instead of `println`

`private static final Logger LOG = LoggerFactory.getLogger(MyStartingEventListener.class);`